sdk/src/Services/S3/Generated/Model/Internal/MarshallTransformations/PutBucketIntelligentTieringConfigurationRequestMarshaller.cs

```diff
@@ -0,0 +1,144 @@
+            PreMarshallCustomization(request, publicRequest);
+            request.HttpMethod = "PUT";
+            request.AddSubResource("intelligent-tiering");
+        
+            if (publicRequest.IsSetExpectedBucketOwner()) 
+            {
+                request.Headers["x-amz-expected-bucket-owner"] = publicRequest.ExpectedBucketOwner;
+            }
+            if (string.IsNullOrEmpty(publicRequest.BucketName))
+                throw new System.ArgumentException("BucketName is a required property and must be set before making this call.", "PutBucketIntelligentTieringConfigurationRequest.BucketName");
+            if (string.IsNullOrEmpty(publicRequest.IntelligentTieringId))
+                throw new AmazonS3Exception("Request object does not have required field IntelligentTieringId set");
+            
+            if (publicRequest.IsSetIntelligentTieringId())
+                request.Parameters.Add("id", StringUtils.FromString(publicRequest.IntelligentTieringId));
+            request.ResourcePath = "/";
+            var stringWriter = new XMLEncodedStringWriter(CultureInfo.InvariantCulture);
+            using (var xmlWriter = XmlWriter.Create(stringWriter, new XmlWriterSettings() { Encoding = System.Text.Encoding.UTF8, OmitXmlDeclaration = true, NewLineHandling = NewLineHandling.Entitize }))
+            {   
+                if (publicRequest.IsSetIntelligentTieringConfiguration())
+                {
+                    xmlWriter.WriteStartElement("IntelligentTieringConfiguration", "http://s3.amazonaws.com/doc/2006-03-01/");
+                    IntelligentTieringFilterCustomMarshall(publicRequest, xmlWriter);
+                    if(publicRequest.IntelligentTieringConfiguration.IsSetIntelligentTieringId())
+                        xmlWriter.WriteElementString("Id", StringUtils.FromString(publicRequest.IntelligentTieringConfiguration.IntelligentTieringId));
+
+                    if(publicRequest.IntelligentTieringConfiguration.IsSetStatus())
+                        xmlWriter.WriteElementString("Status", StringUtils.FromString(publicRequest.IntelligentTieringConfiguration.Status));
+
+                    var publicRequestIntelligentTieringConfigurationTierings = publicRequest.IntelligentTieringConfiguration.Tierings;
+                    if (publicRequestIntelligentTieringConfigurationTierings != null && (publicRequestIntelligentTieringConfigurationTierings.Count > 0 || !AWSConfigs.InitializeCollections)) 
+                    {
+                        foreach (var publicRequestIntelligentTieringConfigurationTieringsValue in publicRequestIntelligentTieringConfigurationTierings) 
+                        {
+                        if (publicRequestIntelligentTieringConfigurationTieringsValue != null)
+                        {
+                            xmlWriter.WriteStartElement("Tiering");
+                            if(publicRequestIntelligentTieringConfigurationTieringsValue.IsSetAccessTier())
```

Comment:
Days used to come before AccessTier. It shouldn't matter in this case but calling it out.